src/components/LibrariesList/LibrariesList.tsx

```diff
@@ -0,0 +1,28 @@
+import './style.css';
+import {LIBRARIES, LIBRARIESLISTHEADER} from "../../utils/constants.ts";
+
+export default function LibrariesList() {
+    
+    return (
+        <table>
+            <thead>
+            <tr>
+                {LIBRARIESLISTHEADER.map((item, index) => (
+                    <th key={index}>{item}</th>
+                ))}
+            </tr>
+            </thead>
+            <tbody>
+            {LIBRARIES.map((item, index) => (
```

Comment:
It would be better not to use index for key, but make it more unique

Reply:
Dear Volha,

Updated: added unique IDs for array elements in constants